src/features/todo/Element.tsx

```diff
@@ -2,20 +2,33 @@ import { EllipsisVerticalIcon } from '@heroicons/react/16/solid';
 import { Todo } from '../../types/Todo';
 import { Checkbox } from '../../components/Checkbox';
 import { DropdownMenu } from '../../components/DropdownMenu';
+import { dateFormatter } from '../../utils/dateFormatter';
 
 type Props = {
   item: Todo;
   onDeleteButtonClick: () => void;
+  onCheckboxClick: () => void;
 };
 
-export const Element = ({ item, onDeleteButtonClick }: Props) => {
-  const { name } = item;
+export const Element = ({ 
+  item,
+  onDeleteButtonClick,
+  onCheckboxClick,
+}: Props) => {
+  const { name, createdAt, done } = item;
+
+  const formattedDate = dateFormatter(createdAt);
 
   return (
     <div className="w-full h-12 bg-white rounded-lg flex justify-between items-center pr-3 relative overflow-hidden">
-      <div className="flex items-center h-full outline-none">
-        <Checkbox />
-        <h3 className="text-stone-700 font-medium">{name}</h3>
+      <div className="flex justify-between items-center h-full w-full outline-none px-1">
```

Comment:
(Suggestion) And adding margin here would be need too:
```suggestion
      <div className="flex justify-between items-center h-full w-full outline-none px-1 mr-1">
```